### ISSUE TITLE
need a new release

### ISSUE THREAD

**Issue by hefengxian** [patrickschur/language-detection#8]
I was using composer install this package, but i still has this error

```
PHP Fatal error:  Uncaught TypeError: array_merge() expects at least 1 parameter, 0 given in /home/.../vendor/patrickschur/language-detection/src/LanguageDetection/NgramParser.php:139
Stack trace:
#0 /home/.../vendor/patrickschur/language-detection/src/LanguageDetection/NgramParser.php(139): array_merge()
#1 /home/.../vendor/patrickschur/language-detection/src/LanguageDetection/Language.php(50): LanguageDetection\NgramParser->getNgrams('\xEF\xBC\x9F\xEF\xBC\x9F')
#2 /home/.../Fetchers/Mapping.php(140): LanguageDetection\Language->detect('\xEF\xBC\x9F\xEF\xBC\x9F')
#3 /home/.../test/index.php(31): Fetchers\Mapping::languageDetect('\xEF\xBC\x9F\xEF\xBC\x9F')
#4 {main}
  thrown in /home/.../vendor/patrickschur/language-detection/src/LanguageDetection/NgramParser.php on line 139
```
but this bug is fixed by this commit [610e8a1](https://github.com/patrickschur/language-detection/commit/610e8a1626575f2eb52e27d81bd000bc96a247e1)

when run `composer update patrickschur/language-detection` the code still not updated

**Comment by patrickschur** [comment]
I fixed this with a new release 3.1.1.
Thanks buddy. :)

**Comment by hefengxian** [comment]
Thanks.
